va/va.h

```diff
@@ -2906,6 +2915,17 @@ typedef struct _VAEncMiscParameterCustomRoundingControl {
         uint32_t    value;
     }   rounding_offset_setting;
 } VAEncMiscParameterCustomRoundingControl;
+typedef struct _VAEncDeltaQPBuffer {
+    /*
+     * \brief This structure holds delta QP per block, the size of block must be queried 
+     * for the specified codec, like HEVC.
+     * Buffer size shall be sufficient to fit the slice or frame to be encoded depending on
+     * if it is a slice level or frame level encoding.
```

Comment:
What order are the blocks in?
I assume if encode order and raster scan order agree then it's that, but they don't match in codecs with tiles (VP9, AV1 or HEVC) when there is more than one tile horizontally.

Reply:
The order is raster scan order. The buffer is filled in with delta qp  based on frame.